### PR TITLE
Update Windows Binary and Fix Readme

### PR DESCRIPTION
**Pull request by vikas5914**
- Update Windows Binary to 1.4.3
- Added missing "," in readme file

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ var args = [
 	'--strip-iptc',
 	'--strip-icc',
 	'--all-progressive',
-	'--dest=build'
+	'--dest=build',
```
Comment (shinnn) [inline readme.md]: Could you create a separate commit for this change? `Add a missing comma to the example script`

**Comment by shinnn** [comment]
:clap:

Could you tell us the detailed process how to build jpegoptim on Windows? Some (or many) Windows users seem to have trouble with building it. https://github.com/tjko/jpegoptim/issues/18

**Comment by shinnn** [comment]
Ah, I just found your [repo](https://github.com/vikas5914/jpegoptim-win32) and [site](https://www.kapadiya.net/jpegoptim-win32/). Really helpful project!

**Comment by vikas5914** [comment]
Delete This pull request .

**Comment by shinnn** [comment]
> Delete This pull request .

Should have been *closed* by yourself.

**Pull request closed without merging**